just_audio/lib/just_audio.dart

```diff
@@ -2131,12 +2141,20 @@ abstract class UriAudioSource extends IndexedAudioSource {
 /// your device to forward HTTP requests with headers included.
 class ProgressiveAudioSource extends UriAudioSource {
   ProgressiveAudioSource(Uri uri,
-      {Map<String, String>? headers, dynamic tag, Duration? duration})
-      : super(uri, headers: headers, tag: tag, duration: duration);
+      {Map<String, String>? headers,
+      dynamic tag,
+      Duration? duration,
+      Map<String, String>? drm})
```

Comment:
Add trailing comma here for mult-line lists - and anywhere else that's affected.